seed/models/columns.py

```diff
@@ -259,6 +260,7 @@ def create_mappings(mappings, organization, user, import_file_id=None):
                 cache_column_mapping.append(
                     {
                         'from_field': mapping['from_field'],
+                        'from_units': mapping.get('from_units'),
```

Comment:
any reason not to conver the other items in this dict to mapping.get('*') to make it consistent?

Reply:
To me, anyway, it's signalling that `from_units` is optional, which I think is the only item for which that's true. Could also give it a name if the explicit optional meaning is important:

```py
new_mapping = {'from_field': mapping['from_field'] ... }
new_mapping['from_units'] = mapping.get('from_units')  # could be missing; depends on mapping
cache_column_mapping.append(new_mapping)
```